### PR TITLE
Initial plugin implementation

### PR DESCRIPTION
**Pull request by christophermaier**
Signed-off-by: Christopher Maier <chris@graplsecurity.com>

### CODE REVIEW

File: integration/auth/.htpasswd [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PyPIUser:$2y$05$c7UBMMsfOEJnwwUOTql5d.7e3ZJTtxMxIdcw7l.5l/lza8SR0mLTK
```
Comment (jgrillo-grapl) [inline integration/auth/.htpasswd:1]: is this a secret?

Comment (jgrillo-grapl) [reply integration/auth/.htpasswd:1]: nvm, I just read the README below :p

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (jgrillo-grapl) [inline .gitpod.yml:1]: is this (and the `.gitpod` directory) intentionally added?

**Review by jgrillo-grapl — APPROVED** [review]

File: .gitpod.yml (hunk 2 above)
Comment (christophermaier) [reply .gitpod.yml:1]: Yeah... I figure this is a minimum we could do to make things easier for external contributors.

**Pull request merged**